From 5eaec60cfd1442e528a884fdac2c5a74537b426d Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Mon, 3 Dec 2018 10:33:41 +0100
Subject: [PATCH] base-files: Make sure umount doesn't break on restart

Just to be on safe side, make sure that restart doesn't do anything when
triggering umount init script.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 package/base-files/files/etc/init.d/umount | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/package/base-files/files/etc/init.d/umount b/package/base-files/files/etc/init.d/umount
index 57a4a6d..b764ae1 100755
--- a/package/base-files/files/etc/init.d/umount
+++ b/package/base-files/files/etc/init.d/umount
@@ -2,6 +2,11 @@
 # Copyright (C) 2006 OpenWrt.org
 
 STOP=90
+
+restart() {
+	:
+}
+
 stop() {
 	sync
 	/bin/umount -a -d -r
-- 
2.22.0

